android/app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +32,7 @@
             android:name=".notifications.NotificationService"
             android:enabled="true"
             android:exported="false"/>
+        <service android:name="com.mapbox.mapboxsdk.telemetry.TelemetryService" />
```

Comment:
Please do, I'm afraid of what legal reasons we will need to add in our side in order to use this.